sqs_workers/sqs_env.py

```diff
@@ -50,35 +50,43 @@ def queue(
         self,
         queue_name,  # type: str
         queue_maker=JobQueue,  # type: Type[AnyQueue]
+        batching_policy=NoBatching(),  # type: BatchingConfiguration
         backoff_policy=None,  # type: Optional[BackoffPolicy]
     ):
-        # type: (...) -> GenericQueue
+        # type: (...) -> AnyQueue
         """
         Get a queue object, initializing it with queue_maker if necessary.
         """
         if queue_name not in self.queues:
             backoff_policy = backoff_policy or self.backoff_policy
             self.queues[queue_name] = queue_maker(
-                env=self, name=queue_name, backoff_policy=backoff_policy
+                env=self,
+                name=queue_name,
+                batching_policy=batching_policy,
+                backoff_policy=backoff_policy,
             )
         return self.queues[queue_name]
 
     def processor(
-        self, queue_name, job_name, pass_context=False, context_var=DEFAULT_CONTEXT_VAR
+        self,
+        queue_name: str,
+        job_name: str,
+        pass_context=False,
+        context_var=DEFAULT_CONTEXT_VAR,
     ):
         """
         Decorator to attach processor to all jobs "job_name" of the queue "queue_name".
         """
-        q = self.queue(queue_name, queue_maker=JobQueue)  # type: JobQueue
+        q: JobQueue = self.queue(queue_name, queue_maker=JobQueue)  # type: ignore
```

Comment:
You can remove `# type: ...` comments if you're using type annotations. We're not trying to support Python 2.7 anymore anyway 🙂

Reply:
Thank goodness for that, I didn't want to rock the boat, but not a fan 😊

`# type: ignore` is intentional here, getting `mypy` warnings as it can't work out that passing `queue_maker=JobQueue` into `self.queue` always makes it return a `JobQueue` type, and the `processor` function called afterwards doesn't exist on the base class (nor should it as `RawQueue` has a different signature)

Reply:
Done, moved all typing to annotations over comments